src/rpc/server.cpp

```diff
@@ -567,8 +569,17 @@ std::string HelpExampleCli(const std::string& methodname, const std::string& arg
 
 std::string HelpExampleRpc(const std::string& methodname, const std::string& args)
 {
-    return "> curl --user myusername --data-binary '{\"jsonrpc\": \"1.0\", \"id\":\"curltest\", "
-        "\"method\": \"" + methodname + "\", \"params\": [" + args + "] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/\n";
+    const int rpcPort = gArgs.GetArg("-rpcport", BaseParams().RPCPort());
+
+    std::ostringstream help;
```

Comment:
Please choose another variable name to avoid shadowing function `help` :-)

Reply:
Fixed, another good catch!  Do you have an automated script to check for those shadowing instances? :)